cloud/extensions.rst

```diff
@@ -0,0 +1,68 @@
+| hstore             | 1.4     | public     | data type for storing sets of (key, value) pairs                   |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| intarray           | 1.2     | public     | functions, operators, and index support for 1-D arrays of integers |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| ltree              | 1.1     | public     | data type for hierarchical tree-like structures                    |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| pg_buffercache     | 1.2     | public     | examine the shared buffer cache                                    |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| pg_freespacemap    | 1.1     | public     | examine the free space map (FSM)                                   |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| pg_prewarm         | 1.1     | public     | prewarm relation data                                              |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| pg_stat_statements | 1.4     | public     | track execution statistics of all SQL statements executed          |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| pg_trgm            | 1.3     | public     | text similarity measurement and index searching based on trigrams  |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| pgcrypto           | 1.3     | public     | cryptographic functions                                            |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| pgrowlocks         | 1.2     | public     | show row-level locking information                                 |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| pgstattuple        | 1.4     | public     | show tuple-level statistics                                        |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| plpgsql            | 1.0     | pg_catalog | PL/pgSQL procedural language                                       |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| session_analytics  | 1.0     | public     |                                                                    |
```

Comment:
Do these two (`session_analytics` and `shard_rebalancer`) not have a description? If not, could you open an issue in each of those repo's to add one? (Or better yet, open up a PR to do so)

Reply:
Sure, I'll send PRs.